browser/ui/config.gni

```diff
@@ -21,6 +21,7 @@ if (is_android) {
 }
 if (toolkit_views) {
   brave_ui_allow_circular_includes_from += [
+    "//brave/browser/ui/views/frame/split_view",
```

Comment:
Hi, @simonhong. I'm trying to split vertical tabs into a separator source_set like thi. I'm curious if adding target to `brave_ui_allow_circular_includs_from` also add the source sets to the actual build target. So, there's no need to add `deps += [frame/split_view]` to `browser/ui/BUILD.gn`?

Reply:
Don't need to add to `brave/browser/ui` target explicitely.
`brave_ui_allow_circular_includs_from` is added to deps of `brave/browser/ui` in `brave/browser/ui/BUILD.gn`

Reply:
Got it. Thanks!